react-components/toolbar-filter/step5.md

```diff
@@ -1,67 +1,120 @@
 
 <pre class="file">
-  &lt;DataToolbarItem variant=&quot;search-filter&quot;&gt;
-    &lt;Input value=&quot;&quot; id=&quot;step2Input&quot; ariaLabel=&quot;Step 2 input&quot; /&gt;
+&lt;DataToolbarItem variant=&quot;search-filter&quot;&gt;
+  &lt;Input value=&quot;&quot; id=&quot;step2Input&quot; ariaLabel=&quot;Step 2 input&quot; /&gt;
+&lt;/DataToolbarItem&gt;
+&lt;DataToolbarGroup variant=&quot;filter-group&quot;&gt;
+  &lt;DataToolbarItem&gt;
+    &lt;CheckboxSelect
+      onSelect={this.onSelect}
+      type=&quot;Status&quot;
+      selections={filters.status}
+      options={statusOptions}
+    /&gt;
   &lt;/DataToolbarItem&gt;
-  &lt;DataToolbarGroup variant=&quot;filter-group&quot;&gt;
-    &lt;DataToolbarItem&gt;
-      &lt;CheckboxSelect
-        onSelect={this.onSelect}
-        type=&quot;Status&quot;
-        selections={filters.status}
-        options={statusOptions}
-      /&gt;
-    &lt;/DataToolbarItem&gt;
-    &lt;DataToolbarItem&gt;
-      &lt;CheckboxSelect
-        onSelect={this.onSelect}
-        type=&quot;Risk&quot;
-        selections={filters.risk}
-        options={riskOptions}
-      /&gt;
-    &lt;/DataToolbarItem&gt;
-  &lt;/DataToolbarGroup&gt;
+  &lt;DataToolbarItem&gt;
+    &lt;CheckboxSelect
+      onSelect={this.onSelect}
+      type=&quot;Risk&quot;
+      selections={filters.risk}
+      options={riskOptions}
+    /&gt;
+  &lt;/DataToolbarItem&gt;
+&lt;/DataToolbarGroup&gt;
 </pre>
 
-2) Wrap both the `DataToolbarItem` and the `DataToolbarGroup` in the same `DataToolbarToggleGroup`. Provide both the `toggleIcon` and `breakpoint` properties to the `DataToolbarToggleGroup`.
+3) **Replace each of the three `DataToolbarItems` inside the `DataToolbarToggleGroup` with a `DataToolbarFilter`**
+
+4) **Add a `chips` property to each of the `DataToolbarFilter`**
+
+Each of the `DataToolbarFilter` components requires a `chips` property. In this case, the state managed filters array's properties can serve as the `chips` property for each `DataToolbarFilter`. 
+
+* In this case, the first `DataToolbarFilter` (containing a text input) should have the property `chips={filters.name}`. 
+* The second `DataToolbarFilter` (containing the status filter) should have the property `chips={filters.status}`. 
+* The third `DataToolbarFilter` (containing the risk filter) should have the property `chips={filters.risk}`.
+
+5) **Pass an appropriate label as the `categoryName` property to each `DataToolbarFilter`**
+
+* In this case, the first `DataToolbarFilter` (containing a text input) should have the property `categoryName="Name"`. 
+* The second `DataToolbarFilter` (containing the status filter) should have the property `categoryName="Status"`. 
+* The third `DataToolbarFilter` (containing the risk filter) should have the property `categoryName="Risk"`.
+
+6) **Pass the onDelete function as the `deleteChip` property to each `DataToolbarFilter`**
```

Comment:
Give an example here of what this looks like:

<DataToolbarFilter deleteChip={this.onDelete}>

Reply:
I have a code snippet after step 6. I will reword the next line to say 'Once steps 4, 5, and 6 are completed, the contents of the `DataToolbarToggleGroup` should appear as follows:"